modules/nf-core/kraken2/build/meta.yml

```diff
@@ -0,0 +1,44 @@
+name: "kraken2_build"
+description: Builds Kraken2 database
+keywords:
+  - metagenomics
+  - db
+  - classification
+  - build
+  - kraken2
+tools:
+  - kraken2:
+      description: "Kraken2 is a system for assigning taxonomic labels to short DNA sequences, usually obtained through metagenomic studies."
+      homepage: https://ccb.jhu.edu/software/kraken2/
+      documentation: https://github.com/DerrickWood/kraken2/wiki/Manual
+      tool_dev_url: "https://github.com/DerrickWood/kraken2"
+      doi: 10.1186/s13059-019-1891-0
+      licence: ["MIT"]
```

Comment:
According to docs, missing an `args_id` entry [^1]? (this is new to me :sweat_smile: ) 

[^1]: https://nf-co.re/docs/contributing/modules#documentation point 5